src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -434,4 +478,16 @@ export class StakingModuleGuardService {
 
     return true;
   }
+
+  /**
+   * @returns true if nonce is the same
+   */
+  public isSameStakingModuleContractState(
+    firstNonce: number,
+    secondNonce: number,
+  ): boolean {
+    if (firstNonce !== secondNonce) return false;
```

Comment:
why not just
`return firstNonce == secondNonce`

Reply:
deleted function